src/test/java/teammates/test/cases/action/InstructorFeedbackQuestionEditActionTest.java

```diff
@@ -1209,24 +1209,12 @@ public void testExecuteAndPostProcessRubricQuestion() {
                 Const.ParamsNames.FEEDBACK_QUESTION_NUMBER, Integer.toString(fq.questionNumber),
                 Const.ParamsNames.FEEDBACK_QUESTION_TYPE, "RUBRIC",
                 Const.ParamsNames.FEEDBACK_QUESTION_TEXT, fqd.getQuestionText() + "(edited)",
-                Const.ParamsNames.FEEDBACK_QUESTION_DESCRIPTION, "more details",
```

Comment:
Why there are changes others beside `FEEDBACK_QUESTION_RUBRIC_WEIGHT`?

Reply:
As @bqnguyen94 asked in https://github.com/TEAMMATES/teammates/pull/8856#discussion_r192153525 if repeated params can be extracted, I did that only. So that, it is a easier to read the test cases.

Reply:
Where do you extract to?

Reply:
Oops! I misinterpreted the comment!

Reply:
@bqnguyen94 @xpdavid Do we need to extract the repeated params?
This is what I came up with:

```java
String[] requiredParams = { ....}; // The required fields that will be added in every test
String[] testRelatedParams = {......};

List<String> params = new ArrayList<>(Arrays.asList(requiredParams));
Collections.addAll(params, editTextParams);

InstructorFeedbackQuestionEditAction a = getAction(params.toArray(new String[params.size()]));
// Rest of the code
```
Is this worth the added advantage?

Reply:
This works. I have no preference on whether you should do this in your PR or not.

Reply:
Yes, I think if the extraction makes the code significantly cleaner then it's worthwhile, unless it takes considerable amount of work to do. The decision is yours 👍